config/cable.docker.yml

```diff
@@ -0,0 +1,7 @@
+redis: &redis
+  adapter: redis
+  url: <%= ENV.fetch("REDIS_URL") { "redis://redis:6379/1" } %>
+
+production:
+  <<: *redis
+
```

Comment:
Please remove newline

Reply:
it's a configuration of editor or IDE, auto adding new-line at the end of the file, please don't spam with such comment, one is enough.